.changeset/rich-hounds-grow.md

```diff
@@ -0,0 +1,13 @@
+---
+"@razorpay/blade": minor
+---
+
+feat: add `Accordion` and `Collapsible` pre-release
+
+- `Accordion` and `Collapsible` are available on web as pre-release
+- You may import and use the following: `Accordion_PRE_RELEASE` and `Collapsible_PRE_RELEASE`
+- The stories are marked internal for now, till the stable release, visit them from [here](https://blade.razorpay.com/?path=/story/guides-intro--page&globals=showInternalComponents:true;measureEnabled:false)
```

Comment:
maybe just enable them? just make the component name as `Accordion_PRE_RELEASE`?

Reply:
Done in https://github.com/razorpay/blade/pull/1277/commits/c70905cb15f43025db797c0785265e3615fbadfc